models/template_helpers_test.go

```diff
@@ -137,6 +137,27 @@ func TestRuleTemplates(t *testing.T) {
 		assert.Equal(t, cParams.Source, updated.Source)
 	})
 
+	t.Run("change err - mismatch names", func(t *testing.T) {
+		tx, err := db.Begin()
+		require.NoError(t, err)
+		defer func() {
+			require.NoError(t, tx.Rollback())
+		}()
+
+		q := tx.Querier
+
+		name := gofakeit.UUID()
+
+		cParams := createTemplateParams(name)
```

Comment:
cParams - Isn't this style of variable name consistency breach? The variable is name is not self-explaining, a comment could clarify that.

Reply:
>consistency breach
Why? 

`cParams` = `create params`

Reply:
At the first sight I understood is as const params. But well... When I used this style of naming variables Aleksi and Nurlan told me that it was not consistent with the code and requested changes.
I personally don't have problem with that, just that I would never though 'c' stands for 'create'. I would've thought const or char or count. But not create.

Reply:
I agree with @yareach that it's better to make it clear. I would suggest to rename it to `cp` if we want to use shorten variable names  or `createParams` if we want make it easy to read, but not something between.